Dockerfile.agnos

```diff
@@ -106,6 +89,9 @@ RUN mkdir -p /tmp/agnos/debs
 COPY ./userspace/debs /tmp/agnos/debs
 COPY ./userspace/hardware_setup.sh /tmp/agnos
 RUN /tmp/agnos/hardware_setup.sh
+# Install libffi6 - needed for raylib
+RUN apt-get -o Dpkg::Options::="--force-overwrite" install -yq \
+    /tmp/agnos/debs/libffi6_3.2.1-8_arm64.deb
```

Comment:
where does this come from? we can't apt install

Reply:
moved libffi6 to libs/